rules/do-you-know-the-benefits-of-using-source-control/rule.md

```diff
@@ -19,17 +19,17 @@ With the source control (we use TFS), we can share project code and cooperate wi
  
 <!--endintro-->
 
-![Figure: View the changes in source control on each individual file](HistoryWindow.jpg)
+![Figure: Right-click a file and select Git | View History to see the changes in source control.](https://user-images.githubusercontent.com/40375803/126916052-26efa3b9-5014-4d1d-a665-db7cab916acf.png)
```

Comment:
Use local images.

See wiki: https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Add-an-Image